src/main/java/uk/gov/justice/digital/hmpps/keyworker/controllers/ControllerAdvice.java

```diff
@@ -24,7 +26,11 @@
 
     @ExceptionHandler(WebClientResponseException.class)
     public ResponseEntity<byte[]> handleException(final WebClientResponseException e) {
-        log.error("Unexpected exception", e);
+        if (e.getRawStatusCode() == 404) {
+            log.info(format("Not found for %s caused by %s", e.getClass().getName(), e.getMessage()));
```

Comment:
since it is slf4j should be:
```
            log.info("Not found for {} caused by {}", e.getClass().getName(), e.getMessage());
```

Reply:
Sorry, should have been clearer.  I was actually expecting a new 
```
    @ExceptionHandler(NotFound.class)
```
block rather than an if block on the parent class

Reply:
maybe you could even add the class as an extra handler on the not found one i.e.
```
    @ExceptionHandler({EntityNotFoundException.class, NotFound.class})
```

Reply:
The not found exception is wrapped in a WebClientResponseException - that's what bubbles up to the ControllerAdvice.  So I don't think I can do this without rethrowing everywhere it appears?

Reply:
Or do you mean rethrow the not found from within ControllerAdvice... hmmm I think that would work.

Reply:
I mean changing the `handleNotFoundException` method to:
```java

    @ExceptionHandler({EntityNotFoundException.class, NotFound.class})
    public ResponseEntity<ErrorResponse> handleNotFoundException(final Exception e) {
        log.info("Not found for {} caused by {}", e.getClass().getName(), e.getMessage());
        return ResponseEntity
                .status(HttpStatus.NOT_FOUND)
                .body(ErrorResponse
                        .builder()
                        .status(HttpStatus.NOT_FOUND.value())
                        .developerMessage(e.getMessage())
                        .build());
    }

```